examples/common/src/main.adb

```diff
@@ -0,0 +1,6 @@
+procedure Main is
```

Comment:
What is this file for? You should declare `common` as `library project` instead so that it does not need a `main`.

Reply:
It shouldn't be there. I'm removing it.
It must have got automatically generated by GPS.
Still, I should have noticed it.

Reply:
`main` deleted.
Bad styles corrected.